src/api/todos.ts

```diff
@@ -5,4 +5,14 @@ export const getTodos = (userId: number) => {
   return client.get<Todo[]>(`/todos?userId=${userId}`);
 };
 
-// Add more methods here
+export const addTodo = (
+  title: string,
+  userId: number,
+  completed = false,
+) => {
```

Comment:
You should pass an object, so you will not make mistake with correct order of params